InputTime/index.jsx

```diff
@@ -36,8 +36,8 @@ const renderAmPm = ({
       onChange={e => onChange({
         ...value,
         hours: e.target.value === 'AM'
-        ? value.hours - 12
-        : value.hours + 12,
+        ? parseInt(value.hours, 10) - 12
+        : parseInt(value.hours, 10) + 12,
       })}
```

Comment:
I wonder if converting the `string` a `number`/`int` would be better to do outside of the component? In the `propTypes` for `InputTime`  we set the type to `number` so passing a string would throw a warning:

https://github.com/bufferapp/buffer-components/pull/100/files#diff-5549b598a5fa977558e56361acd6a28fL138

With this change it will still throw a warning but will handle a `string` properly. But I think it would better to keep the component clean and handle conversion on the outside.